debezium-connector-db2/src/main/java/io/debezium/connector/db2/ChangeTable.java

```diff
@@ -0,0 +1,99 @@
+/*
+ * Copyright Debezium Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.debezium.connector.db2;
+
+import io.debezium.relational.TableId;
+
+/**
+ * A logical representation of change table containing changes for a given source table.
+ * There is usually one change table for each source table. When the schema of the source table
+ * is changed then two change tables could be present.
+ *
+ * @author Jiri Pechanec
+ *
+ */
+public class ChangeTable {
```

Comment:
Ok, that's copied from the SQL Server module, right? Let's extract it into core then and have a single version (general comment, we should have some shared infra for these polling-based connectors instead of duplicating it).